docs/self-managed/identity/user-guide/making-identity-production-ready.md

```diff
@@ -0,0 +1,23 @@
+---
+id: making-identity-production-ready
+title: "Making Identity production ready"
+sidebar_label: "Making Identity production ready"
+---
+
+We recommend considering the following topics when moving Identity into a production environment.
```

Comment:
Another thing that must be specified in production mode is `IDENTITY_URL`, otherwise the login will not work